docs/user-guide/background.md

```diff
@@ -49,6 +49,23 @@ In particular, from this window, the user can perform the following operations:
 
 Once the options are chosen, with the <img src="../img/button/add_group_confirm_button.jpg" class="ms-docbutton"/> button the new background layer is definitively added to the background selector as a card and automatically set as the current one.
 
+### Add WMTS background
+
+As soon as a layer, from a [WMTS Catalog](catalog.md#wmswmts-catalog), is added to the list of backgrounds, and it is selected, the **Add New Background window** opens: 
```

Comment:
In case of a WMTS layer added as a background layer, the **Add New Background window** is a bit different: